### ISSUE TITLE
[Feat] Add an option to edit previous comment instead of posting a new one

### ISSUE THREAD

**Issue by jBouyoud** [suzuki-shunsuke/github-comment#473]
Hello,

It might be very interesting to add an option that allows edit previous comment instead of hide/posting a new one.

History is kept by Github on each comment.

Regards,
I'll be glad to help implementing this is you think that is relevant.

**Comment by suzuki-shunsuke** [comment]
# Idea 1: Add the field `update`

This is just an idea.

e.g.

```yaml
post:
  default:
    update: 'Comment.HasMeta && Comment.Meta.TemplateKey == "default"'
    template: hello
```

1. Update the comment matches with the condition. If multiple comments match, the latest comment is updated
1. If no comment matches, aa new comment is created

**Comment by jBouyoud** [comment]
LGTM

Would you like I tried to open a PR to implement this spec ? Or did you prefer do it by your own ?

**Comment by suzuki-shunsuke** [comment]
Thank you for your contribution!
I've published the prerelease version v4.3.0-0.
https://github.com/suzuki-shunsuke/github-comment/releases/tag/v4.3.0-0

We'll implement the same feature against `github-comment exec` command.
